### ISSUE TITLE
Future inclusion on the linux kernel

### ISSUE THREAD

**Issue by josgramaxo** [lwfinger/rtl8723bu#112]
Greetings.
I have been using the out of tree driver every day for 2 or 3 months. It works great. Thank You. Are any plans ofinclusion of the driver on the linux kernel?

**Comment by lwfinger** [comment]
No. Reworking this driver to satisfy kernel coding standards would require at least 6 months full time, and more likely it would be more like a full year. I'm too old to waste that much of my life. The device is handled by driver rtl8xxxu, which is cleanly coded. Any effort would be better spent in determining why the crappy Realtek driver has better performance that rtl8xxxu does.

The same argument is true for every one of the Realtek USB drivers. All the baggage needed to support both Linux, FreeBSD, and Windows makes for a bloated, unintelligible mess!
